engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -416,11 +416,18 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
     import keys._
 
     val DataStoreUpdate(runnableKeys, updatedData) = data.executionStoreUpdate
-    val runnableCalls = runnableKeys.view collect { case k if k.node.isInstanceOf[CallNode] => k } sortBy { k =>
-      (k.node.fullyQualifiedName, k.index.getOrElse(-1)) } map { _.tag }
-
+    val runnableCalls = runnableKeys.view
+      .collect({ case k: BackendJobDescriptorKey => k })
+      .groupBy(_.node)
+      .map({
+        case (node, keys) => 
+          val tag = node.fullyQualifiedName
+          val shardCount = keys.map(_.index).distinct.size
+          if (shardCount == 1) tag
+          else s"$tag ($shardCount shards)"
+      })
     val mode = if (restarting) "Restarting" else "Starting"
-    if (runnableCalls.nonEmpty) workflowLogger.info(s"$mode calls: " + runnableCalls.mkString(", "))
+    if (runnableCalls.nonEmpty) workflowLogger.info(s"$mode " + runnableCalls.mkString(", "))
```

Comment:
Interesting that the old code was sorting and the new code is not and no tests seem to mind. I guess we got rid of that yucky integrationesque test from the early days?

Reply:
yeah I was expecting tests to fail to with this change but none seemed to care. I actually wanted to draw some attention on this, we were previously logging every single call that was started here, including every shard and its number. This just give the number of shards instead, also do not differentiate attempts.
Are we ok with losing some of this info in the logs ?